libcodechecker/server/run_db_model.py

```diff
@@ -103,7 +104,8 @@ class File(Base):
     __tablename__ = 'files'
 
     id = Column(Integer, autoincrement=True, primary_key=True)
-    filepath = Column(String)
+    filepath = Column(String, nullable=False)
+    filename = Column(String, nullable=False)
```

Comment:
Does it make sense to store the filename separately? Don't we have some functionality to compute this on demand (on the SQL server)? E.g.: see https://stackoverflow.com/questions/19505875/parse-file-name-and-path-from-full-path